exporters/trace/jaeger/README.md

```diff
@@ -40,27 +40,27 @@ For Maven add to your `pom.xml`:
   <dependency>
     <groupId>io.opencensus</groupId>
     <artifactId>opencensus-api</artifactId>
-    <version>0.13.0</version>
+    <version>0.12.2</version>
```

Comment:
I don't think this needs to be updated, since Jaeger exporter is not included in v0.12.2 release, instead it's planned to be released in v0.13.0.

Reply:
Ack.